w3af/plugins/auth/autocomplete_js.py

```diff
@@ -256,22 +273,71 @@ def _find_form_submit_strategy(self, chrome, form):
 
         return None
 
-    def has_active_session(self, debugging_id=None):
+    def has_active_session(self, debugging_id=None, chrome=None):
         """
         Check user session with chrome
         """
         has_active_session = False
         self._set_debugging_id(debugging_id)
-        chrome = self._get_chrome_instance(load_url=False)
+        if not chrome:
+            chrome = self._get_chrome_instance(load_url=False)
 
         try:
             chrome.load_url(self.check_url)
-            chrome.wait_for_load()
+            loaded = chrome.wait_for_load()
+            if not loaded:
+                msg = 'Failed to load %s in chrome for autocomplete_js'
+                args = (self.check_url,)
+                self._log_debug(msg % args)
+                return False
             has_active_session = self.check_string in chrome.get_dom()
         finally:
             chrome.terminate()
             return has_active_session
 
+    def get_options(self):
+        """
+        :returns OptionList: list of option objects for plugin
+        """
+        option_list = super(autocomplete_js, self).get_options()
+        autocomplete_js_options = [
+            (
+                'username_field_css_selector',
+                self.username_field_css_selector,
+                STRING,
+                "(Optional) Exact CSS selector which will be used to retrieve "
+                "the username input field. If provided then w3af won't try "
```

Comment:
In most cases I was trying NOT to mention `w3af` in things that the end-user might see. If you guys ever expose this to the user, they will know that w3af is used in the backend.

Maybe change to: `When provided the scanner is not going to try to detect the input field in an automated way`

Same problem below.